Update campaign_builder.rst - Added Transactional No DNC text to M5 branch (#136)
mautic/user-documentation: docs/campaigns/campaign_builder.rst

     - Sends a Webhook to a defined URL, using the GET, POST, PUT, PATCH, or DELETE methods. Headers and data is customizable, and support the use of tokens, such as Contact fields and the Contact's IP address. For example, ``{contactfield=firstname}``
   * - **Send Email**   

>      - Sends a transaction or marketing Email to the selected Contact. You can send a transactional Email to the Contact multiple times. You can only send a marketing Email to the Contact  once across multiple sources. If the Contact has already received this Email from another source or the current Campaign, they aren't sent the Email again and the Contact progresses through the Campaign.

Reviewer: ```suggestion
```

>      - Sends a transaction or marketing Email to the selected Contact. You can send a transactional Email to the Contact multiple times. You can only send a marketing Email to the Contact  once across multiple sources. If the Contact has already received this Email from another source or the current Campaign, they aren't sent the Email again and the Contact progresses through the Campaign.

Reviewer 2: @RCheesley I'm not quite understand about the content. But from your responses here, can we link the KB article (https://kb.mautic.org/article/understand-the-difference-transactional-and-marketing-emails-in-mautic.html) here like so:

```suggestion
* - **Send Email**   
  - Sends a transactional or marketing Email to the selected Contact. Learn more about transactional and marketing Email in (this article).
```

Reviewer 2: If that's okay, I can help to apply it.

Reviewer: Yes, I think that's fine, using a link macro

>      - Emails can be transactional or marketing. Marketing emails will obey Email Channel's Do Not Contact setting of the contact, Transactional Emails will disregard the Do Not Contact value. This allows to send Transactional emails as campaign step to Contacts even if they opted out to recieve Marketing emails via standard unsubscribe action and have DNC set as 'no'. Enable transactional email ignoring DNC can be turned on in Email Settings.

Reviewer: ```suggestion
     - Sends a transaction or marketing Email to the selected Contact. You can send a transactional Email to the Contact multiple times. You can only send a marketing Email to the Contact once across multiple sources. If the Contact has already received this Email from another source or the current Campaign, they aren't sent the Email again and the Contact progresses through the Campaign. Emails can be transactional or marketing. Marketing Emails obey Do Not Contact (DNC) settings of the Contact, Transactional Emails disregard the DNC value. This allows sending Transactional Emails as a Campaign step to Contacts, even if they opted out to receive marketing Emails via a standard unsubscribe action and therefore have DNC set for the Email Channel. Turned on by default in Mautic 5.0, it's possible to override the setting in Email Settings.
```

Just like we test code PR's in Gitpod, please review your docs edits in Gitpod too! This one broke the RST table layout, and there were several Vale fixes to do after addressing that 😸 .

There should be a Gitpod button to click and you'll see any issues in the 'output' tab.

As a side thought, this is a big wall of text to have in a small area in a table. Maybe it makes more sense to have a dedicated section which explains transactional and marketing emails, and link to it from this with a small description? WDYT @jos0405 ? I think a sub-section of emails.rst under Channels would make the most sense.
